code/src/WijmoProvider/Columns/DropdownColumn.ts

```diff
@@ -128,9 +135,12 @@ namespace WijmoProvider.Column {
             const dataMap = this.config.dataMap;
             const values = dataMap.collectionView.items;
 
-            const column = this.grid.getColumn(this.config.parentBinding);
+            const parentColumn = this.grid.getColumn(this.config.parentBinding);
```

Comment:
What if this.config.parentBinding is undefined, what's the expected result from the getColumn?

Reply:
Undefined, but I check for it on the if below

Reply:
Ok, I was concerned about the result of the getColumn when the parentBinding was undefined.